homeassistant/components/uvc/camera.py

```diff
@@ -92,6 +92,17 @@ def name(self):
         """Return the name of this camera."""
         return self._name
 
+    @property
+    def supported_features(self):
+        """Return supported features."""
+        caminfo = self._nvr.get_camera(self._uuid)
```

Comment:
This call does I/O. We're not allowed to do I/O in entity state properties.  We need to fetch the info before the entity is created and pass it as an argument to the entity on instantiation.